data-engineering/How-to-use-conception-tickets.s.md

```diff
@@ -0,0 +1,26 @@
+# [Standard] How to tackle tickets with conception phase (3 min)
+
+### Owner: [Quentin Febvre](https://github.com/quentinf00)
+
+
+## Why:
+
+ - To detect problem earlier during the implementation of a ticket
+ - To ensure vsibility on what is done during the implementation of a ticket
+
+This can be a tool to reduce the lead time on features with uncertainties, the conception phase can be used to prioritize kanban ticket into you sprint
+
+
+## Checks
```

Comment:
Sur les control points tu peux mettre des emojis comme ici : http://blog.m33.network/2016/09/prepare-your-ceremony-to-make-it-short-and-tactac/

Reply:
Oki :)